modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -286,10 +286,6 @@ public void processOpts() {
 
         supportingFiles.add(new SupportingFile("exceptions.mustache", packagePath(), "exceptions.py"));
 
-        if (Boolean.FALSE.equals(excludeTests)) {
-            supportingFiles.add(new SupportingFile("__init__.mustache", testFolder, "__init__.py"));
-        }
-
```

Comment:
Why did we remove this?

Reply:
Since we don't need the test folder to be a package. I can re-add it, if you want.

Reply:
If you could revert this, that would be great. Everything else looks good.

Reply:
@michelealbano this if statement needs to be added back in